rj_gameplay/rj_gameplay/tactic/pass_tactic.py

```diff
@@ -73,6 +71,16 @@ def create_request(self, **kwargs) -> role.RoleRequest:
         """
         pass
 
+    def find_potential_receiver(self, world_state: rc.WorldState) -> rc.Robot:
+        cost = float('inf')
+        receive_robot = None
+        for robot in world_state.our_robots:
+            self.receiver_cost(robot, None, world_state) < cost
```

Comment:
I think you should make a local var here to avoid calling the same method 3 times back-to-back.

Also, this is where I did my find-and-replace for the crashing `self.pass_cost` line. Please lmk if I shouldn't have: you can revert that last commit with `git revert <commit-hash>`